### PR TITLE
Implements form tag for the quiz card: Rebased

### PR DESCRIPTION
**Pull request by mayamolodec**
This commit substitutes the main div tag of the quiz-card to
the form tag and changes button.onClick to form.onSubmit
event. This is a new version with reviews fixed.

### CODE REVIEW

File: scripts/main.js [hunk 1]
```diff
@@ -189,8 +187,13 @@ function checkAnswer(){
 
     }
 
+    response.preventDefault();
```
Comment (w1n732mu73) [inline scripts/main.js]: Please don't suppress side effects of form submission here. Do it inside your form event listener initialization
```
addEventListener('submit', (e) => {
     e.preventDefault()
     ...
   }
)
```

**Review by w1n732mu73 — CHANGES_REQUESTED** [review]

File: scripts/main.js (hunk 1 above)
Comment (mayamolodec) [reply scripts/main.js]: Okay, I'll fix it!

**Comment by mayamolodec** [comment]
New commit moves submit event function to eventListener initialization
and changes the way we find the checked radio button: now we do it with 
FormData().

File: scripts/main.js [hunk 2]
```diff
@@ -39,8 +39,38 @@ let question_index = 0;
...
 const wrapContainer = document.querySelector('#wrap');
 
-let quizCard = document.createElement('div');
+let quizCard = document.createElement('form');
 quizCard.className="quiz";
+quizCard.nam = "answer";
+quizCard.addEventListener("submit", (e)=>{
+    e.preventDefault();
+    const formData = new FormData(quizCard);
+    const checkedRadio = formData.get('answer');
```
Comment (w1n732mu73) [inline scripts/main.js]: There is no neeed in formData. It should look like this
```suggestion
    const checkedRadio = e.target.answer.value

```

**Review by w1n732mu73 — CHANGES_REQUESTED** [review]

File: scripts/main.js (hunk 2 above)
Comment (mayamolodec) [reply scripts/main.js]: Ah, okay. I thought with the target there could be some 
event bubbling problem. I will fix it now.

**Comment by mayamolodec** [comment]
This commit changes how we search the checked
radio button: we use target now. |

**Review by w1n732mu73 — APPROVED** [review]

**Pull request merged**